geometry_msgs/Vector3 angular_velocity		# gyro, degrees/sec
geometry_msgs/Vector3 linear_acceleration	# acc, m/s^2
geometry_msgs/Vector3 compass_orientation	# compass, some unit
